analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -163,19 +113,6 @@ module "eks_blueprints_kubernetes_addons" {
     })]
   }
 
-  enable_aws_load_balancer_controller = true
-  aws_load_balancer_controller = {
-    version = "1.4.7"
-    timeout = "300"
-  }
-
-  enable_ingress_nginx = true
-  ingress_nginx = {
-    version = "4.5.2"
-    timeout = "300"
-    values  = [templatefile("${path.module}/helm-values/nginx-values.yaml", {})]
-  }
```

Comment:
These two are being used for setting up Spark Live UI using path based routing for each Spark Job. Keep these two but add some comments that it is added for building Spark Live UI with Spark Operator config.

Can we get the output of the NLB DNS Name from this add-on and add this here https://github.com/awslabs/data-on-eks/blob/ad33f0331cd793c25b64768a822a6ee358a2ade8/analytics/terraform/spark-k8s-operator/helm-values/spark-operator-values.yaml#L42?

Reply:
re-added